### DISCUSSION TITLE
Can you add windowed fullscreen option for your amazing patcher?

### DISCUSSION THREAD

**Discussion by 130N1D** [FaithBeam/Sims-1-Complete-Collection-Widescreen-Patcher#44] [Ideas]
I know that we can add -w option to the shortcut, and then use side software as Windowed Borderless Gaming, but it would be cool to add this feature to your patch.

**Comment by FaithBeam** [comment]
@130N1D Hello! Thanks for the compliment! This is a great idea, but I have almost zero experience using the win32 api, and using that api from c# is kind of a pain. I'm also assuming that's how people make applications borderless because I haven't tried. The drawback with this method is that you'd have to keep the patcher open to update the Sims window, and at that point you may as well use the well tested Windowed Borderless programs. I'll look into it, but I have a few things I'd like to do before that.

**Comment by 130N1D** [comment]
I noticed this feature in Smooth Patch from Lazy Duchess for The Sims 3, it has option to make application borderless, check it out:
https://lazyduchess.tumblr.com/post/657467259898527744/the-sims-3-smooth-patch-12

**Reply by FaithBeam** [reply]
When I get the chance I'll take a look, thank you.

**Comment by 130N1D** [comment]
I also noticed when I play in windowed mode when and move the camera there are these graphical artifacts like this:
https://sun9-48.userapi.com/impg/ZyHb7fz0PHgHal-W9q7Enws10VXnhIvSNw58iA/osmAKRjsRGY.jpg?size=2560x1440&quality=96&sign=dd1cb06da3e594ba8782383fd23860a5&type=album
I play with resolution 2560x1920 and DDrawCompact wrapper.

**Reply by FaithBeam** [reply]
Yea I have the same issue with ddrawcompat when playing at high resolutions. This isn't an issue with my code, someone would have to open an issue in https://github.com/narzoul/DDrawCompat

**Comment by FaithBeam** [comment]
@130N1D I don't know if you've tried this but I can get something like borderless windowed by installing the patch as the same resolution as the desktop and appending -w to the target box for your Sims executable.

https://youtu.be/uLisY-zwWW0

**Reply by 130N1D** [reply]
Thank you! Works for me.
